src/basic_commands/write.md

```diff
@@ -12,43 +12,37 @@ Write bytes using the `w` command. It accepts multiple input formats like inline
 
 ```
 [0x00404888]> w?
```

Comment:
Maybe we don't need this full output here, hard to maintain

Reply:
many things needs rework